modules/claims_api/app/sidekiq/claims_api/claim_establisher.rb

```diff
@@ -42,27 +42,16 @@ def perform(auto_claim_id) # rubocop:disable Metrics/MethodLength
 
       queue_flash_updater(auto_claim.flashes, auto_claim_id)
       queue_special_issues_updater(auto_claim.special_issues, auto_claim)
-    rescue ::EVSS::DisabilityCompensationForm::ServiceException => e
+    rescue ::Common::Exceptions::ServiceError => e
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
-      auto_claim.evss_response = e.messages
+      auto_claim.evss_response = e.errors || e.detailed_message
       auto_claim.form_data = orig_form_data
       auto_claim.save
-      log_exception_to_sentry(e)
-    rescue ::Common::Exceptions::BackendServiceException => e
-      ClaimsApi::Logger.log('claims_establisher',
-                            retry: false,
-                            detail: "/submit failure for claimId #{auto_claim&.id}: #{e.original_body}, #{e.class}")
-      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
-      auto_claim.evss_response = [{ 'key' => e.status_code, 'severity' => 'FATAL', 'text' => e.original_body }]
-      auto_claim.form_data = orig_form_data
-      auto_claim.save
-      log_exception_to_sentry(e)
     rescue => e
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
-      auto_claim.evss_response = e.detailed_message
+      auto_claim.evss_response = (e.errors.presence || e.detailed_message)
```

Comment:
It is saving here the one time now, but I am wondering if the response is getting changed and if that is something we can change?  By changed I mean different keys in the returned message. I assumed it needed to maintain the same format of having a key and everything, but perhaps not?

This is what I see sending in a `::Common::Exceptions::BackendServiceException`
![Screenshot 2024-03-05 at 10 23 00 AM](https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/ef3bee40-98cd-4276-b01e-7613ba51fe5d)

I thought, based on the old behavior, that it would have been formatted like (had the same keys I mean)
<img width="471" alt="Screenshot 2024-03-05 at 10 34 23 AM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/4cc91b70-2465-4e5d-951f-3cda324e37c6">

Reply:
@rockwellwindsor-va I want to make sure I understood correctly. I changed the hash keys to match other error objects, and now they look like the following:
``` e.errors
[{"title"=>"Client error", "detail"=>"A client exception has occurred, job will not be re-tried. original_error: .", :status=>"400", :code=>"400"}]```
When it is a VA900, I set the code equal to that as a string.